webknossos-datastore/Dockerfile

```diff
@@ -16,5 +16,11 @@ RUN chown -R webknossos . \
 
 USER webknossos
 
+HEALTHCHECK \
+  --interval=1m --timeout=5s --retries=10 \
+  CMD curl --fail http://localhost:9090/data/health || exit 1
```

Comment:
Really weird that docker allows the exit-code to be only 0 or 1.

Reply:
Yeah, that's weird.